bot/tests/administration/test_list_admins.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from bot.tests.base_test import BaseTest
+
+
+class TestListAdminsCommand(BaseTest):
+
+    @pytest.mark.quick
+    def test_list_admins_with_admins(self):
+        response = self.send_command('/listadmins')
+        expected_fragments = ["2015344951"]
```

Comment:
nie wiem do konca o co tu chodzi, ale testy generalnie powinny byc tabula rasa xD

przygotowujesz srodowisko (np. wrzucasz tego admina)
operujesz na przygotowanym srodowisku w tescie
na koniec testu resetujesz srodowisko

Reply:
bo teraz zakladasz ze z jakiegos powodu bedzie admin (chyba ze robi to base test w jakis sposob), co jest krzywa praktyka

Reply:
Są dosyć tabula rasa i z resetem stanowiska bo zakładam że mam psute /mk do testów i po każdym teście je czyszczę XD

Reply:
Ale to i tam więcej komentarzy właśnie nawiązujesz do tego podpięcia bazy i generalnie +1 ale zacząłem od testu czy dana komenda wgl działa i tak sobie wykombinowałem żeby robić te testy dokładniejsze już pojedynczo jak połatam komendy

Reply:
A co do podłączania bazy to użyć kodu z bota co mamy ? Czy te testy powinny być niezależne od bota ? I tak samo pytanie to tych settingsow które skopiowałem nwm na ile te testy powinny korzystać z kodu bota bo na to zrobiłem totalnie odseparowane

Reply:
hmm no settingsy jakies envy to testy powinny miec faktycznie osobne, ale jak juz beda jakies testy potwierdzajace ze sam kod bazy dziala poprawnie to w sumie czemu z tego nie skorzystac

ewentualnie po prostu w bazowym fixture trzasnac drop wszystkich tabel i zaladowanie jakiegos podstawowego konfigu, to pare linijek powinno byc zeby setup/teardown czyscil baze kompletnie